data/hai/hai reveal 1 intro.txt

```diff
@@ -255,7 +256,7 @@ mission "ZZ Event Hai Secret Leaks: FWH"
 mission "ZZ Event Hai Secret Leaks: RS"
 	landing
 	name "Investigate the Hai"
-	description "A secret and violent alien race that traders have known about for ages? Preposterous; go find out what's really going on."
+	description "There have been reports of a secret, violent alien race that traders have known about for ages. The rumours seem unsubstantiated, but you should go and find out what's really going on."
```

Comment:
Descriptions overrun the text box if they're too long. I wouldn't extend this unless you first check that it fits in-game.

Reply:
I got rid of "The rumours seem unsubstantiated"; and anyway it's an example of telling, not showing.